openapi-bal-service/src/main/java/io/ballerina/openapi/converter/service/OpenAPIComponentMapper.java

```diff
@@ -405,7 +405,7 @@ private Schema mapEnumValues(EnumSymbol enumSymbol) {
         List<ConstantSymbol> enumMembers = enumSymbol.members();
         for (ConstantSymbol enumMember : enumMembers) {
             if (enumMember.typeDescriptor().typeKind() == TypeDescKind.SINGLETON) {
-                enums.add(enumMember.typeDescriptor().signature());
+                enums.add(enumMember.typeDescriptor().signature().replaceAll("\"", ""));
```

Comment:
This logic might fail when there are escaped double quotes present within the string literal itself. 
(e.g: `string foo = "Hello \"World\""`). So shall we add a test case for a similar scenario?

Reply:
Yeah will add the test for it.